tools/proofers/spellcheck.inc

```diff
@@ -218,10 +220,12 @@ if ($user->profile->i_type == 1) {
 </form>
 </div>
 <?php
-
+if ($user->profile->i_type != 1) {
```

Comment:
we don't have constants for the i_type values we could use? A genuine question, but it'd be nice if we could have some define we reference.

Reply:
If we are not planning to deploy this but only use it as a step along the way to a new PI this stuff will all go away.